fr/taxonomie_utilisateurs.md

```diff
@@ -0,0 +1,14 @@
+Taxonomie des utilisateurs
+==========================
```

Comment:
Ok, there's already mixed markup on that, I'll make a separate PR for unifying it afterwards  :)